clippy_lints/src/misc.rs

```diff
@@ -169,7 +169,7 @@ impl<'tcx> LateLintPass<'tcx> for LintPass {
                         TOPLEVEL_REF_ARG,
                         arg.hir_id,
                         arg.pat.span,
-                        "`ref` directly on a function argument is ignored. \
+                        "`ref` directly on a function argument requires ownership of the argument nonetheless. \
```

Comment:
"does not prevent taking ownership of the passed parameter"

Reply:
Better.

I suggest: "`ref` directly on a function parameter does not prevent taking ownership of the passed argument" which is stricter in terms of param/arg.